src/lib.rs

```diff
@@ -0,0 +1,210 @@
+use lambda_extension::Extension;
```

Comment:
We should add the short copyright statement to this source file, like the main.rs file.

```
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
```